Exam/CT/src/exercice2/MaskedRaster.java

```diff
@@ -0,0 +1,33 @@
+package exercice2;
+
+public class MaskedRaster {
+	public static int[][] bits;
+	public static boolean[][] exclusionMask;
+	
+	private MaskedRaster(int[][] input) {
+		bits = input;
+		initMask();
+	}
+	
+	private void initMask() {
+		//exclusionMask = ;
```

Comment:
:sob: fill exclusionMask with false? You've done something similar in exercice1!!!